connector/src/main/scala/quasar/qscript/Coalesce.scala

```diff
@@ -544,7 +564,21 @@ class CoalesceT[T[_[_]]: BirecursiveT: OrderT: EqualT: ShowT] extends TTypes[T]
                   case RightSide => rmf.as[JoinSide](RightSide)
                 }
                 EquiJoin(ej.src, innerLSrc, innerRSrc, newLKey, newRKey, ej.f, newCombine).some
-              case _ => none
+              case (\/-(Some(Map(innerLSrc, lmf))), _) =>
```

Comment:
if I remember correctly, this wasn't fixing anything in particular but was just a thing we might be able to do